.gitignore

```diff
@@ -2,4 +2,6 @@
 */test/.git-sources
 */test/.tilt-cache
 */test/.build
-*/test/tilt_modules
\ No newline at end of file
+*/test/tilt_modules
+
+workflow_*.tmp
```

Comment:
i also talked to Lucie about this, but I would really like to see these stored in the tilt apiserver as configmaps rather than as random files on disk

Reply:
Yeah - that'd also help it not re-init itself w/o using an env var to hold state. I'll play around with that later and open another PR